src/utils/zod/schemas/jobParametersSchema.ts

```diff
@@ -12,9 +14,23 @@ export const swapUpdateAdditionalParamsSchema = newAdditionalParamsSchema.extend
 });
 
 export const updateAdditionalParamsSchema = swapUpdateAdditionalParamsSchema.extend({
-  displayPath: z.string().uuid(),
+  displayPath: displayPathSchema,
 });
 
+export const catalogUpdateAdditionalParamsSchema = z
```

Comment:
implement new interface/type for CatalogUpdateAdditionalParams with the optional fields like "displayPath"